src/main/java/it/pagopa/pn/deliverypush/service/impl/WebhookEventsServiceImpl.java

```diff
@@ -196,4 +202,44 @@ private Set<String> statusByVersion(int version) {
             .map(NotificationStatusInt::getValue)
             .collect(Collectors.toSet());
     }
+
+    private Set<String> categoriesByFilter(StreamEntity stream) {
+        Set<String> categoriesSet;
+        if (stream.getFilterValues()== null || stream.getFilterValues().isEmpty()) {
+            categoriesSet = defaultCategories;
+        } else {
+            categoriesSet = stream.getFilterValues().stream()
+                    .filter(v -> !v.equalsIgnoreCase(DEFAULT_CATEGORIES))
+                    .collect(Collectors.toSet());
+            if (stream.getFilterValues().contains(DEFAULT_CATEGORIES)) {
+                categoriesSet.addAll(defaultCategoriesPa);
+            }
+        }
+        return categoriesSet;
+    }
+
+    private Set<String> getCategoriesPa() {
```

Comment:
Per leggibilità, aver semplicemente una configurazione con la lista delle categories in formato CSV, così poi spring le carica direttamente in una lista?

Reply:
Sì, esattamente, le categories saranno caricate direttamente in una lista della classe PnDeliveryPushConfigs;
la modifica richiesta è stata effettuata.